common/landed_titles/wc_religious_titles.txt

```diff
@@ -3,11 +3,11 @@
 @always_primary_score = 1000
 
 #########################
-#		Religion Head		#
+#	Religion Head       #
 #########################
 
-###air_group
```

Comment:
I mean the religion name is important too.

Reply:
readded in 67226e6e0